package.json

```diff
@@ -42,17 +43,18 @@
     "eslint-config-prettier": "^8.3.0",
     "eslint-plugin-header": "^3.1.1",
     "eslint-plugin-import": "^2.22.1",
-    "eslint-plugin-jest": "^26.0.0",
+    "eslint-plugin-jest": "^27.1.1",
     "eslint-plugin-prettier": "^4.0.0",
```

Comment:
You can remove most, if not all of these, they now come from the shared config package.

Reply:
Done in 8e4102341a4f85e56e788421c4e7d5491754bb78 (also wow a lot of linting things overdue)